hub/executors/indexers/vector/annoy/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM jinaai/jina:devel
+
+RUN apt-get update && apt-get install --no-install-recommends -y g++
+RUN pip install annoy
+
+RUN pip install jina
```

Comment:
`jina` is already included in the base image.
```suggestion
```

Reply:
hadn't realized I'm installing the image again. removed it.